test/TemperatureControlTest.cpp

```diff
@@ -0,0 +1,65 @@
+#include <Arduino.h>
+#include <ArduinoUnitTests.h>
+#include <ci/ObservableDataStream.h>
+
+#include "TankControllerLib.h"
+#include "TemperatureControl.h"
+
+/*
+bundle exec arduino_ci.rb --skip-examples-compilation --testfile-select=TemperatureControlTest.cpp
```

Comment:
Is this needed? It seems like a common command that a developer should know. Would it make more sense to have a section of the read me explaining how to run a single test file?

Reply:
The more things I don't have to remember the better; I can apply my scarce and failing memory to other things. This line shows how to run a single test and that's something that I know can be done but I don't remember how to do it. I'd rather not have to go to a README.md to piece this command line together from a page of instructions and options and arguments every time I need it.

Reply:
Ok, can you add a comment above to make it clear what this line does and a note its a shortcut for developers.